doc/source/cluster/kubernetes/examples/tpu-serve-stable-diffusion.md

```diff
@@ -0,0 +1,63 @@
+(kuberay-tpu-stable-diffusion-example)=
+
+# Serve a Stable Diffusion model on GKE with TPUs
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples). This example is adapted from the [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/tools/ray_tpu/src/serve) Cloud TPU example.
+
+## Step 1: Create a Kubernetes cluster with TPUs
+
+Follow [Creating a GKE Cluster with TPUs for KubeRay](kuberay-gke-tpu-cluster-setup) to create a GKE Autopilot or Standard cluster with 1 CPU node and 1 TPU node.
+
+## Step 2: Install the KubeRay operator
+
+Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator via Helm repository. Multi-host TPU support is provided in KubeRay v1.1.0+. Please note that the YAML file in this example uses `serveConfigV2`, which is supported starting from KubeRay v0.6.0.
+
+## Step 3: Install the RayService
+
+```sh
+# Creates a RayCluster with a single-host v4 TPU worker group of 2x2x1 topology
+kubectl apply -f https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-service.tpu-single-host.yaml
+```
+
+KubeRay operator v1.1.0 adds a new `NumOfHosts` field to the RayCluster CRD, supporting multi-host worker groups. This field specifies the number of workers to create per replica, with each replica representing a multi-host PodSlice. The value for `NumOfHosts` should match the number of TPU VM hosts expected by the given `cloud.google.com/gke-tpu-topology` node selector. For this example, the Stable Diffusion model is small enough to run on a single TPU host, so `numOfHosts` is set to 1 in the RayService manifest.
+
+## Step 4: View the Serve deployment in the Ray dashboard
+
+Verify the RayService is deployed and running
+
+```sh
+kubectl get rayservice
+
+# NAME               SERVICE STATUS   NUM SERVE ENDPOINTS
+# stable-diffusion-tpu-serve-svc   Running          2
+```
+
+Port-forward the Ray dashboard from the Ray head service. To view the dashboard, open http://localhost:8265/ on your local machine.
+
+```sh
+kubectl port-forward svc/stable-diffusion-tpu-head-svc 8265:8265 &
+```
+
+You may now monitor the status of the RayService in the Ray dashboard from the the 'Serve' tab. The installed RayService
+should create a running application with the name 'stable_diffusion'. The Ray Serve application will have two Ray Serve deployments, the
+API ingress which will receive input prompts and the Stable Diffusion model server.
+
```

Comment:
how about adding a screenshot of the dashboard here?

Reply:
Done in [f5d27f4](https://github.com/ray-project/ray/pull/45924/commits/f5d27f4112c93262a135572a45a36b30b365faa2), I also updated some of the instructions (there was a step missing to create a python virtual environment and install the needed dependencies to run the prompt script).